mungegithub/mungers/mungerutil/util.go

```diff
@@ -16,14 +16,137 @@ limitations under the License.
 
 package mungerutil
 
-import "github.com/google/go-github/github"
+import (
+	"strings"
 
-// FilterValid filters a list of user data and returns only user data that contains valid username.
-func FilterValid(users []*github.User) (res []*github.User) {
-	for _, u := range users {
-		if u != nil && u.Login != nil {
-			res = append(res, u)
+	"github.com/google/go-github/github"
+)
+
+// ActiveUsers tracks Users involved in a github Issue
+type ActiveUsers struct {
+	assignees []string
+	author    string
+}
+
+func mergeUsers(userslists ...[]string) []string {
+	uniqueUsers := map[string]struct{}{}
+
+	// Fill-in map of all the users in each lists (remove duplicates)
+	for _, userslist := range userslists {
+		for _, user := range userslist {
+			uniqueUsers[user] = struct{}{}
+		}
+	}
+
+	// Get the list of unique users
+	users := []string{}
+	for user := range uniqueUsers {
+		users = append(users, user)
+	}
+	return users
+}
+
+// GetUser returns nil if the user is invalid, Login name otherwise
+func GetUser(user *github.User) *string {
+	if user == nil {
+		return nil
+	}
+	return user.Login
+}
+
+// GetUsersList returns the list of valid users
+func GetUsersList(users []*github.User) []string {
+	allUsers := []string{}
+
+	for _, user := range users {
+		validUser := GetUser(user)
+		if validUser != nil {
+			allUsers = append(allUsers, *validUser)
+		}
+	}
+	return mergeUsers(allUsers)
+}
+
+// CreateActiveUsersFromIssue make a new object from a github.Issue
+func CreateActiveUsersFromIssue(issue *github.Issue) *ActiveUsers {
```

Comment:
My very last nit. The name `GetActive..` would be easier to read from the outside in my opinion. Also, the comment above is a little ambiguous.